src/LinuxTracing/PerfEventOpen.cpp

```diff
@@ -229,7 +229,7 @@ int tracepoint_with_stack_event_open(const char* tracepoint_category, const char
   pe.sample_type |= PERF_SAMPLE_REGS_USER | PERF_SAMPLE_STACK_USER | PERF_SAMPLE_RAW;
   pe.sample_regs_user = SAMPLE_REGS_USER_ALL;
 
-  pe.sample_stack_user = stack_dump_size;
+  pe.sample_stack_user = 2048;
```

Comment:
There should probably be some explanation regarding this. And probably a TODO regarding cleaning it up (wrt the `stack_dump_size` parameter for example)?